src/clients/security.rb

```diff
@@ -89,16 +89,16 @@ def main
           }
         },
         "options"    => {
-          "home"        => {
-            # command line help text for 'level home' option
+          "workstation"        => {
+            # command line help text for 'level workstation' option
             "help" => _(
-              "Home Workstation security level (without network)"
+              "Workstation security level"
             )
           },
-          "network"     => {
-            # command line help text for 'level network' option
+          "roaming"     => {
+            # command line help text for 'level roaming' option
```

Comment:
Translators will not understand what it means ... roaming is in fact a new term for them